homeassistant/components/netatmo/climate.py

```diff
@@ -213,13 +218,12 @@ def __init__(
 
         self._device_name = self._data.rooms[home_id][room_id]["name"]
         self._attr_name = f"{MANUFACTURER} {self._device_name}"
-        self._current_temperature: float | None = None
-        self._target_temperature: float | None = None
-        self._preset: str | None = None
+        self._attr_current_temperature: float | None = None
+        self._attr_target_temperature: float | None = None
+        self._attr_preset_mode: str | None = None
         self._away: bool | None = None
-        self._operation_list = [HVAC_MODE_AUTO, HVAC_MODE_HEAT]
         self._support_flags = SUPPORT_FLAGS
-        self._hvac_mode: str = HVAC_MODE_AUTO
+        self._attr_hvac_mode: str = HVAC_MODE_AUTO
```

Comment:
Already typed...
```suggestion
        self._attr_hvac_mode = HVAC_MODE_AUTO
```

Could also be set as a class attribute